site/content/docs/user/quick-start.md

```diff
@@ -344,6 +344,27 @@ nodes:
 - role: worker
 ```
 
+#### Limit node resources
+`kind` implements [docker resource constraints][docker resource constraints],
+so you can limit the memory and  CPU used by any cluster node.
+
+```yaml
+kind: Cluster
+apiVersion: kind.sigs.k8s.io/v1alpha3
+nodes:
+# the control plane node
+- role: control-plane
+  constraints:
+    memory: "200m"
+    cpu: "2"
+- role: worker
```

Comment:
This example should probably constrain all nodes, just drop this node.

Reply:
I was trying to demonstrate that is optional and you can do it per node but seems it can be misleading